components/ui/ActionableModal.tsx

```diff
@@ -1,38 +1,44 @@
 import React, { useState } from 'react'
 import styled from 'styled-components'
-import { TinaReset } from '@tinacms/styles'
-import { Modal, ModalPopup, ModalBody } from 'tinacms'
-import { AuthLayout } from '../layout'
-import { Button } from './Button'
+import { TinaReset, Button as TinaButton } from '@tinacms/styles'
+import {
+  Modal,
+  ModalPopup,
+  ModalBody,
+  ModalActions,
+  ModalHeader,
+} from 'tinacms'
 
 export interface ActionableModalOptions {
   title: string
   message: string
-  actions: Array<{ name: string; action(): void }>
+  actions: Array<{ name: string; action(): void; primary?: boolean }>
 }
 
 export const ActionableModal = ({
   title,
   message,
   actions,
 }: ActionableModalOptions) => {
-  const buttons = actions.map(individualAction => (
-    <Button color="primary" onClick={individualAction.action}>
-      {individualAction.name}
-    </Button>
-  ))
+  if (!open) {
+    return null
+  }
 
   return (
     <TinaReset>
       <Modal>
         <ModalPopup>
+          <ModalHeader close={null}>{title}</ModalHeader>
```

Comment:
Can you do this?
```suggestion
          <ModalHeader>{title}</ModalHeader>
```
If not we should fix that in `@tinacms/react-modals`

Reply:
Nope looks like it's required

Reply:
https://github.com/tinacms/tinacms/issues/950

Reply:
PR to address issue: https://github.com/tinacms/tinacms/pull/951